packages/server/test/e2e/8_read_only_fs.ts

```diff
@@ -0,0 +1,55 @@
+import * as fs from 'fs'
+import * as path from 'path'
+
+const e2e = require('../support/helpers/e2e')
+const chmodr = require('chmodr')
+
+describe('e2e project dir access', function () {
+  e2e.setup()
+  const projectDir: string = path.join('/', 'tmp', '8_read_only_fs')
+
+  beforeEach(() => {
+    if (fs.existsSync(projectDir)) {
+      return
+    }
+
+    // write out the project structure
+    // we cant sue one define by source code since we need to modify the file permissions on the dir and files
+    fs.mkdirSync(path.join(projectDir))
+
+    fs.writeFileSync(path.join(projectDir, 'cypress.json'), `{"projectId": "2pz86o"}`)
+    fs.mkdirSync(path.join(projectDir, 'cypress'))
+    fs.mkdirSync(path.join(projectDir, 'cypress', 'integration'))
+    fs.writeFileSync(path.join(projectDir, 'cypress', 'integration', 'spec.js'), `it('test',() => expect(true).to.be.true )`)
+    fs.mkdirSync(path.join(projectDir, 'cypress', 'plugins'))
+    fs.writeFileSync(path.join(projectDir, 'cypress', 'plugins', 'index.js'), `module.exports = () => {}`)
+    fs.mkdirSync(path.join(projectDir, 'cypress', 'support'))
+    fs.writeFileSync(path.join(projectDir, 'cypress', 'support', 'index.js'), `module.exports = () => {}`)
+    fs.mkdirSync(path.join(projectDir, 'cypress', 'videos'))
```

Comment:
hmm, I don't think you actually need to do this project setup manually, check out my comment on `e2e.it` for a simpler way

Reply:
Done! Thanks :)